test/compflags/albrecht/chplenv/generateChplEnvFiles

```diff
@@ -11,6 +11,10 @@ generateGood() {
 
     good=""
     for var in ${printchplenv}; do
+        # Ignore "CHPL_GPU = *" because we don't have a compiler flag for that.
+        if [[ "${var}" == "CHPL_GPU="* ]]; then
+            continue
+        fi
```

Comment:
What's wrong with the above `grep -v BLA` way of handling printchplenv's output? You'd need `CHPL_GPU=` instead of `CHPL_GPU` (to avoid ruling out others), but it seems like it's already doing something like what you're trying to achive here.

Reply:
Oh you're right, should have just used grep.

> You'd need CHPL_GPU= instead of CHPL_GPU (to avoid ruling out others),

What do you mean by this?
Wouldn't `grep -v CHPL_GPU` do what we want?

Reply:
I think it would also match `CHPL_GPU_ARCH` etc -- if we want to filter all variables with the `CHPL_GPU` prefix, that works too.

Reply:
I think it makes sense to just match `CHPL_GPU=` for now since this test doesn't care about other stuff right now.